src/collision.cpp

```diff
@@ -231,8 +231,9 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 {
 	static bool time_notification_done = false;
 	Map *map = &env->getMap();
+	ServerEnvironment *s_env = dynamic_cast<ServerEnvironment*>(env);
 
-	ScopeProfiler sp(g_profiler, "collisionMoveSimple()", SPT_AVG);
+	ScopeProfiler sp(g_profiler, s_env ? "collisionMoveSimple() server" : "collisionMoveSimple() client", SPT_AVG);
```

Comment:
suggestion:
```cpp
#define PROFILER_NAME(text) (s_env ? ("Server: " text) : ("Client: " text))
```
and use that

Reply:
I'd still want "collisionMoveSimple()" first, so that they show near each other in other.

Reply:
Also note there are two. Later in the code there's "... collect boxes"

Reply:
I seem to remember that the other profiler stuff has the client/server first, which is why I suggested that.
Anyway this suggestion will work for both occasions.